providers/kit/examples_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/go-kit/log"
-	"github.com/grpc-ecosystem/go-grpc-middleware/providers/kit/v2"
+	"github.com/grpc-ecosystem/go-grpc-middleware/providers/kit"
```

Comment:
Import grouping. Also are we concerned that users will be confused by this not having v2 in its path?

Reply:
Yea, this is weird. I mentioned all in README. The alternative is to start sub modules with v2, which is also weird as `v2` at the end not in the middle like in the core module. So whatever we do, users will have to get used to multi-modules. Once you understand them it's much easier.

We can always release all modules and submodule with exactly the same version, that's an option.  (:

Reply:
Yeah, I agree I think we can keep this as is. It is a new module name and doesn't need a v2.